integration/client/client.go

```diff
@@ -377,8 +380,11 @@ type Rules struct {
 }
 
 // RunRangeQuery runs a query and returns an error if anything went wrong
-func (c *Client) RunRangeQuery(query string) (*Response, error) {
-	buf, statusCode, err := c.run(c.rangeQueryURL(query))
+func (c *Client) RunRangeQuery(ctx context.Context, query string) (*Response, error) {
+	ctx, cancelFunc := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
Since we're defining `30*time.Second` in a few places, maybe it should be a central config of some kind?

Reply:
I made it a const instead then to make it less error prone. WDYT?